docs/user/ibmcloud/tested_instance_types_x86_64.md

```diff
@@ -1,15 +1,16 @@
 * `bx2-8x32`
 * `bx2d-4x16`
 * `bx3d-4x20`
+* `bx3dc-8x40`
 * `cx2-8x16`
 * `cx2d-4x8`
 * `cx3d-8x20`
+* `cx3dc-4x10`
 * `gx2-8x64x1v100`
 * `gx3-16x80x1l4`
 * `mx2-8x64`
 * `mx2d-4x32`
-* `mx3d-2x20`
-* `ox2-4x32`
```

Comment:
Do we no longer support these types?

Reply:
replace  `mx3d-2x20` with 'mx3d-4x40 '
replace `ox2-4x32` with `ox2-8x64`
1. list one type in the same profiles
2. use the larger to support the master and workers.